realm/src/androidTest/java/io/realm/RealmTest.java

```diff
@@ -1800,13 +1800,14 @@ public void testMutableMethodsOutsideTransactions() throws JSONException, IOExce
     // TODO: re-introduce this test mocking the ReferenceQueue instead of relying on the GC
```

Comment:
Yes, i tested that one on my machine and it takes a bit longer to run.
The testing is being re-written by @emanuelez in #1359